crates/cheatnet/src/state.rs

```diff
@@ -317,13 +318,12 @@ impl TraceData {
     }
 }
 
-fn match_node_response<T: Default>(result: StateResult<T>) -> StateResult<T> {
+fn match_node_response<T>(result: StateResult<T>) -> StateResult<T> {
```

Comment:
convert_node_response? but it'd still not be very clear to me what's the purpose of this function through that name

Reply:
This is basically `map_err` if the error is other than `UndeclaredClassHash`

Reply:
I will convert this into `map_err`

Reply:
i actually removed it overall, not needed

Reply:
awesome